app/src/main/java/com/example/una/fragments/HomePageFragment.java

```diff
@@ -32,34 +55,101 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
     @Override
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
-        verticalRecyclerView = view.findViewById(R.id.rvCharitiesVertical);
+        verticalRecyclerView = view.findViewById(R.id.homeRecyclerView);
         // makes verticalRecyclerView's height and width immutable no matter what is inserted or removed inside the recyclerView
         verticalRecyclerView.setHasFixedSize(true);
         verticalRecyclerView.setLayoutManager(new LinearLayoutManager(getContext(), LinearLayoutManager.VERTICAL, false));
         arrayListVertical = new ArrayList<>();
         adapter = new VerticalRecyclerViewAdapter(getContext(), arrayListVertical);
         verticalRecyclerView.setAdapter(adapter);
+        categories = new ArrayList<>();
+
+        client = new AsyncHttpClient();
         setData();
     }
 
     private void setData() {
-        for (int index = 0; index <= 5; index++) {
-            VerticalModel verticalModel = new VerticalModel();
-            verticalModel.setTitle("Title: " + index);
-            ArrayList<HorizontalModel> arrayListHorizontal = new ArrayList<>();
-
-            for (int j = 0; j < 5; j++) {
-                HorizontalModel horizontalModel = new HorizontalModel();
-                horizontalModel.setDescription("Description" + j);
-                horizontalModel.setName("Name: " + j);
-                arrayListHorizontal.add(horizontalModel);
-            }
-            verticalModel.setArrayList(arrayListHorizontal);
+        // featured charities
+        HomeFragmentSection featuredModel = new HomeFragmentSection();
+        featuredModel.setTitle("Featured");
```

Comment:
Can you move the user facing strings to `strings.xml`? This will help you with supporting other languages by adding a new `strings.xml` for that language